includes/SiteGen/SiteGen.php

```diff
@@ -154,6 +155,38 @@ private static function get_patterns_for_category( $category, $site_classificati
 		return $processed_patterns;
 	}
 
+	/**
+	 * Get the templates for a particular category.
+	 *
+	 * @param string $category The category to get templates for.
+	 */
+	private static function get_templates_for_category( $category ) {
+		$response = wp_remote_get(
+			NFD_PATTERNS_BASE . 'templates?category=' . $category,
+			array(
+				'headers' => array(
+					'Content-Type' => 'application/json',
+				),
+				'timeout' => 60,
+			)
+		);
+
+		if ( wp_remote_retrieve_response_code( $response ) !== 200 ) {
+			return array(
+				'error' => __( 'We are unable to process the request at this moment' ),
```

Comment:
Should we pass `wp-module-ai` as the second argument to the `__` function?

Reply:
Done.